app/helpers/application_helper.rb

```diff
@@ -2,4 +2,11 @@ module ApplicationHelper
   def get_article_types
```

Comment:
Yea I just thought it was better practice to have that logic in the controller even though it is small. I can remove the function and put the call for the article types straight in the view though if it is better?